django_coverage_plugin/plugin.py

```diff
@@ -162,6 +163,15 @@ def file_tracer(self, filename):
     def file_reporter(self, filename):
         return FileReporter(filename)
 
+    def find_unexecuted_files(self, src_dir):
+        res = []
+        for i, (dirpath, dirnames, filenames) in enumerate(os.walk(src_dir)):
+            for filename in filenames:
+                if re.match(r"^[^.#~!$@%^&*()+=,]+\.html?$", filename):
```

Comment:
Why the complicated regex?  Yes, this would slightly be slower, but also more readable:

```
ignored, ext = os.path.splitext(filename)
if ext in (".htm", ".html"):
    ...
```